Content.Server/AlertLevel/AlertLevelSystem.cs

```diff
@@ -116,6 +116,16 @@ public float GetAlertLevelDelay(EntityUid station, AlertLevelComponent? alert =
         return alert.CurrentDelay;
     }
 
+    public string GetDefaultLevel(EntityUid station, AlertLevelComponent? alert = null)
```

Comment:
```suggestion
    public string GetDefaultLevel(Entity<AlertLevelComponent?> station)
```
adjust below accordingly